Marlin/src/pins/hc32f4/env_validate.h

```diff
@@ -22,8 +22,8 @@
 #ifndef ENV_VALIDATE_H
 #define ENV_VALIDATE_H
 
-#ifndef ARDUINO_ARCH_HC32
-  #error "Oops! Select an HC32F460 board in 'Tools > Board.'"
+#if NOT_TARGET(ARDUINO_ARCH_HC32)
+  #error "Oops! Make sure you have a HC32F460 environment selected."
```

Comment:
It's a bit antiquated and doesn't apply to most 32-bit / PIO builds, but we should probably stick with the previous error wording for consistency.

Reply:
reverted to the old error wording